spec/cypress/plugins/index.js

```diff
@@ -12,11 +12,11 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+const cucumber = require("cypress-cucumber-preprocessor").default;
```

Comment:
Beautiful!

Reply:
None of it looks beautiful 😂 

Can we `import` it?

Reply:
I was referring to using cucumber - heh you have no appreciation for modern art @kicferk1

Reply:
We cannot `import` it in this file apparently